showcase/src/app/welcome/pages/getting-started/getting-started-page.component.ts

```diff
@@ -14,55 +14,32 @@ import {
-	public runningProject: string;
-	public buildFiles: string;
-	public watchMode: string;
-	public unitTestRunning: string;
-	public watchAndRunTests: string;
-	public runEndToEndTests: string;
-	public continuousIntegrationTests: string;
-	public installType: string;
-	public moduleDeclaration: string;
-	public prototyping: string;
-	public moduleImport: string;
 
-	public ngOnInit(): void {
-		this.installingProject =
-			"# --depth 1 removes all but one .git commit history\n" +
-			"git clone --depth 1 https://github.com/NationalBankBelgium/stark.git";
-		this.runningHMR = "npm run server:dev:hmr";
-		this.runningProject = "npm run server:prod";
-		this.buildFiles =
-			"# development\n" +
-			"npm run build:dev\n\n" +
-			"# production (jit)\n" +
-			"npm run build:prod\n\n" +
-			"# AoT\n" +
-			"npm run build:aot";
-		this.watchMode = "npm run watch";
-		this.unitTestRunning = "npm run test";
-		this.watchAndRunTests = "npm run watch:test";
-		this.runEndToEndTests =
-			"# update Webdriver (optional, done automatically by postinstall script)\n" +
-			"npm run webdriver:update # cfr #35\n\n" +
-			"# this will start a test server and launch Protractor\n" +
-			"npm run e2e";
-		this.continuousIntegrationTests = "# this will test both your JIT and AoT builds\n" + "npm run ci";
-		this.installType = "npm install @types/node\n" + "npm install @types/lodash";
-		this.moduleDeclaration = 'declare module "my-module" {\n' + "\texport function doesSomething(value: string): string;\n" + "}";
-		this.prototyping = "declare var assert: any;\n" + "declare var _: any;\n" + "declare var $: any;";
-		this.moduleImport = 'import * as _ from "lodash";';
+	public installingProject =
+		"# --depth 1 removes all but one .git commit history\ngit clone --depth 1 https://github.com/NationalBankBelgium/stark.git";
+	public runningHMR = "npm run server:dev:hmr";
+	public runningProject = "npm run server:prod";
+	public buildFiles = "# development\nnpm run build:dev\n\n# production (jit)\nnpm run build:prod\n\n# AoT\nnpm run build:aot";
+	public watchMode = "npm run watch";
+	public unitTestRunning = "npm run test";
+	public watchAndRunTests = "npm run watch:test";
+	public runEndToEndTests =
+		"# update Webdriver (optional, done automatically by postinstall script)\nnpm run webdriver:update # cfr #35\n\n# this will start a test server and launch Protractor\nnpm run e2e";
+	public continuousIntegrationTests = "# this will test both your JIT and AoT builds\nnpm run ci";
+	public installType = "npm install @types/node\nnpm install @types/lodash";
+	public moduleDeclaration = 'declare module "my-module" {\n\texport function doesSomething(value: string): string;\n}';
+	public prototyping = "declare var assert: any;\ndeclare var _: any;\ndeclare var $: any;";
+	public moduleImport = 'import * as _ from "lodash";';
 
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public loggingService: StarkLoggingService) {}
+
+	public ngOnInit(): void {
```

Comment:
I think we can safely remove the `constructor` and `ngOnInit` because they are no relevant